jaeseon/app.js

```diff
@@ -49,6 +48,48 @@ app.post('/users', async (req, res) => {
   return res.status(201).json({ message: 'userCreated' });
 });
 
+app.post('/posts', async (req, res) => {
+  const { title, content, user_id } = req.body;
+  await myDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      user_id
+    ) VALUES (?, ?, ?);
+    `,
+    [title, content, user_id]
+  );
+  return res.status(201).json({ message: 'postCreated' });
+});
+
+app.get('/posts', async (req, res) => {
+  const usersPost = await myDataSource.query(
```

Comment:
```suggestion
  const posts = await myDataSource.query(
```

usersPost라는 것은 변수명만 보았을 땐 특정 유저의 게시글 같은 느낌이 있습니다. 지금 SQL문 자체는 특정 유저의 게시글 목록을 조회하기보다는 단순 posts 와 users를 결합하여 posts정보를 조회하는 것이기 때문에 변수명을 명확히 할 필요가 있어보여요